File: 06 - Java/08-02-00-Ejemplo-FuncionSumar-ULP.java
import java.util.Scanner;

public class ProgramaPrincipal {
    
    // Definimos la funcion suma
    static int sumar(int a, int b) {
        int resultadoSuma = a + b;
        return resultadoSuma;
    }

    public static void main(String[] args){
        var consola = new Scanner(System.in);
        System.out.print("Proporciona el valor del primer argumento: ");
        var argA = Integer.parseInt(consola.nextLine());
        System.out.print("Proporciona el valor del segundo argumento: ");
        var argB = Integer.parseInt(consola.nextLine());
        // Llamamos a la funcion sumar
        var resultado = sumar(argA,argB);
        System.out.println("Resultado de la suma: " + resultado);
    }
}

File: 03 - C++/08-02-00-Ejemplo-FuncionSumar-ULP.cpp
#include <iostream>
using namespace std;

// Definimos la funcion suma
int sumar(int a, int b) {
	int resultadoSuma = a + b;
	return resultadoSuma;
}

int main() {
	int argA, argB, resultado;
	// Solicitamos los valores de los argumentos
	cout << "Proporciona el valor del primer argumento: ";
	cin >> argA;
	cout << "Proporciona el valor del segundo argumento: ";
	cin >> argB;
	// Llamamos a la funcion sumar
	resultado = sumar(argA,argB);
	//cout << "Resultado de la suma: " << resultado;
	cout << "Resultado de la suma: " << sumar(argA, argB);
	
	return 0;
}

